test/e2e/upgrade/upgrade.go

```diff
@@ -47,6 +48,7 @@ func NoTests() []upgrades.Test {
 // AllTests includes all tests (minimal + disruption)
 func AllTests() []upgrades.Test {
 	return []upgrades.Test{
+		&adminack.UpgradeTest{},
```

Comment:
I don't know if the order here is important, but I'd like this test to run at the end of the suite (I'd have expected to have that happen via a `[Late]` annotation).  But the `alert.UpgradeTest{}` precedent makes it seem like order does matter here.  I'm not entirely sure how these are positioned relative to the update itself, but the logging makes them seem like they're currently running early in the job:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/26446/pull-ci-openshift-origin-release-4.8-e2e-aws-upgrade/1438885249361645568/artifacts/e2e-aws-upgrade/openshift-e2e-test/artifacts/e2e.log | grep -n 'Cluster should remain functional during upgrade\|admin ack' | grep -B100 passed
84:started: (0/5/5) "[sig-arch][Feature:ClusterUpgrade] Cluster should remain functional during upgrade [Disruptive] [Serial]"
95:[It] Cluster should remain functional during upgrade [Disruptive] [Serial]
167:STEP: Setting up admin ack test
170:Sep 17 16:10:42.260: INFO: Skipping admin ack test. Admin ack is not in this baseline.
1751:passed: (2h47m1s) 2021-09-17T18:57:41 "[sig-arch][Feature:ClusterUpgrade] Cluster should remain functional during upgrade [Disruptive] [Serial]"
```

(with line 170 presumably being written quite a while before line 1751).  Running later (post update) also protects 4.7 -> 4.8 jobs from having `Upgradeable=False` alerting go off, because the openshift/release#22368 approach will not protect those jobs (and some jobs run slow parallel-conformance suites after completing the update).